### ISSUE TITLE
Automation of gbif-impact tool

### ISSUE THREAD

**Issue by djtfmartin** [CatalogueOfLife/backend#1306]
As we plan to support regular (monthly) matching to newer versions of the extended CoL and other checklists, it would be good to be able to trigger on demand or run periodically.

**Comment by mdoering** [comment]
Yes. I had planned to be able to configure CLB projects to call some external handlers/webhooks, probably with a small payload, whenever a new release is done. If we could trigger an impact tool build through some URL that would be great
